### PR TITLE
Added new section - Artificial Intelligence

### PR DESCRIPTION
**Pull request by Nandan-18**
## What does this PR do?
Add resource(s) | Add info | Improve repo

### Why is this valuable (or not)?

The courses combine theory with practical exercises and can be completed at your own pace.

### How do we know it's really free?

A free series of courses made to encourage as broad a group of people as possible to learn what AI is.

### For book lists, is it a book? For course lists, is it a course? etc.

It is a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books.md [hunk 1]
```diff
@@ -735,6 +735,7 @@
 * [freeCodeCamp](https://www.freecodecamp.org)
 * [FutureLearn](https://www.futurelearn.com)
 * [MIT OCW](http://ocw.mit.edu)
+* [MOOC.fi](https://www.mooc.fi/en/)
```
Comment (davorpa) [inline books/free-programming-books.md:738]: Coud you remove this?

You submit it in:
- #5567

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 * [Algorithms & Data Structures](#algorithms--data-structures)
 * [Android](#android)
+* [Artificial Intelligence](#ai)
```
Comment (davorpa) [inline courses/free-courses-en.md:5]: TOC needs be synchronized with their respective section. So use right GH markdown syntax

```suggestion
* [Artificial Intelligence](#artificial-intelligence)
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -372,6 +378,7 @@
 * [Java for Mobile Devices - Introducing Codename One](https://codenameone.teachable.com/p/java-for-mobile-devices-introducing-codename-one)
 * [Java Online Training \| Edureka](https://www.youtube.com/watch?v=hBh_CC5y8-s) - (Edureka)
 * [Java Programming](https://testautomationu.applitools.com/java-programming-course/) - Angie Jones (Applitools)
+* [Java Programming](https://java-programming.mooc.fi) - (University of Helsinki)
```
Comment (davorpa) [inline courses/free-courses-en.md:381]: could you remove this?

You also submit it at:
- #5568

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -119,6 +120,11 @@
 * [Programming Mobile Services for Android Handheld Systems: Concurrency](https://www.coursera.org/course/posaconcurrency)
 
 
+### Artificial Intelligence
+
+* [Elements of AI](https://www.elementsofai.com) - (Reaktor)
```
Comment (davorpa) [inline courses/free-courses-en.md:125]: - Follow creators format guidelines :wink:
- Give credit to all creators. If there are too much, `et al.` could be used as suffix to make the pointcut.

```suggestion
* [Elements of AI](https://www.elementsofai.com) - University of Helsinki, Reaktor
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Please, try to avoid submit incremental Pull Requests, which produce merge conflicts.

So, is time to solve suggestions to scope them. Also some minor changes about format.

File: books/free-programming-books.md (hunk 1 above)
Comment (Nandan-18) [reply books/free-programming-books.md:738]: Hey, I'm a beginner trying to learn about git and GitHub. How do I remove this ?

Comment (davorpa) [reply books/free-programming-books.md:738]: 1. Work on your forked branch: https://github.com/Nandan-18/free-programming-books/blob/my-branch3/books/free-programming-books.md
2. Find file / line and remove it.
3. Commit. Changes will be automatically reflected in PR. like you see what happens with the other suggestions.

File: courses/free-courses-en.md (hunk 3 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Do the same https://github.com/EbookFoundation/free-programming-books/pull/5569#discussion_r718393858 with this.

File: https://github.com/Nandan-18/free-programming-books/blob/my-branch3/courses/free-courses-en.md

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -119,6 +120,11 @@
 * [Programming Mobile Services for Android Handheld Systems: Concurrency](https://www.coursera.org/course/posaconcurrency)
 
 
+### Artificial Intelligence
+
+* [Elements of AI](https://www.elementsofai.com) - University of Helsinki, Reaktor
```
Comment (eshellman) [inline courses/free-courses-en.md:125]: It's usually not helpful to add a category for a single resource. Try to find another category.

**Comment by eshellman** [comment]
Closing, not because it's bad, but because we moved stuff underneath it so it's easier to start from scratch.

**Pull request closed without merging**